python/triton/code_gen.py

```diff
@@ -989,6 +1042,8 @@ def kernel(x_ptr, x_size, **META):
     """
     def decorator(fn):
         def wrapper(kernel):
+            if prune_configs_by:
+                return Autotuner(kernel, fn.arg_names, configs, key, reset_to_zero, prune_configs_by)
             return Autotuner(kernel, fn.arg_names, configs, key, reset_to_zero)
```

Comment:
What is the point of this, I thought `prune_configs_by` defaulted to None anyway?

Reply:
I have updated this